File: SaleBooks.java
public class SaleBooks {
    private int ID;
    private String title;
    private String author;
    private int year;
    private double price;
    private int inventory;
    private boolean available;
    //Método Constructor de la clase.
    public SaleBooks(int ID, String title, String author, int year, double price, int inventory) {
        this.ID = ID;
        this.title = title;
        this.author = author;
        this.year = year;
        this.price = price;
        this.inventory = inventory;
        this.available = true;
    }
    //Getters y Setters de la clase.
    public int getID() {
        return ID;
    }

    public String getTitle() {
        return title;
    }
    public void setTitle(String title) {
        this.title = title;
    }
    public String getAuthor() {
        return author;
    }
    public void setAuthor(String author) {
        this.author = author;
    }
    public int getYear() {
        return year;
    }
    public void setYear(int year) {
        this.year = year;
    }
    public int getInventory() {
        return inventory;
    }
    public void setInventory(int inventory) {
        this.inventory = inventory;
    }
    public double getPrice() {
        return price;
    }
    public void setPrice(double price) {
        this.price = price;
    }
    public boolean isAvailable() {
        return available;
    }
    public void setAvailable(boolean available) {
        this.available = available;
    }
}

File: SaleBooks.py
class SaleBooks:
    def __init__(self, ID, title, author, year, price, inventory):
        self.ID = ID
        self.title = title
        self.author = author
        self.year = year
        self.price = price
        self.inventory = inventory
        self.available = True

    def get_id(self):
        return self.ID

    def get_title(self):
        return self.title

    def set_title(self, title):
        self.title = title

    def get_author(self):
        return self.author

    def set_author(self, author):
        self.author = author

    def get_year(self):
        return self.year

    def set_year(self, year):
        self.year = year

    def get_inventory(self):
        return self.inventory

    def set_inventory(self, inventory):
        self.inventory = inventory

    def get_price(self):
        return self.price

    def set_price(self, price):
        self.price = price

    def is_available(self):
        return self.available

    def set_available(self, available):
        self.available = available